docs/source/federation-spec.md

```diff
@@ -413,3 +413,25 @@ type User @key(fields: "id") {
   name: String @override(from: "SubgraphA")
 }
 ```
+### `@inaccessible`
+```graphql
+directive @inaccessible(from: String!) on FIELD_DEFINITION | INTERFACE | OBJECT | UNION | ARGUMENT_DEFINITION | SCALAR | ENUM | ENUM_VALUE | INPUT_OBJECT | INPUT_FIELD_DEFINITION
+```
+
+The `@inaccessible` directive is used to indicate that a location within the schema is inaccessible. Inaccessible types and fields are available internally but are not exposed publically.
```

Comment:
maybe add a note why you would want to do that -> e.g. something along the lines `Inaccessible can be used to incrementally add schema elements (e.g. fields) to multiple subgraphs without breaking composition.` or maybe something about being useful for surfacing additional data for local debugging, some other use cases?

Reply:
Does this part of the documentation center around _subgraph library authors_ since it's in the `Subgraph specification` part of the docs? We do call out the incremental usecase in the `Federated directives` section https://www.apollographql.com/docs/federation/federated-types/federated-directives/#inaccessible.

Not saying we shouldn't add the note, but also wanted to bring this up (as it was a little confusing for me too).

Reply:
Personally I'd include duplicate information just so folks don't have to jump around multiple pages to get complete info.